tendrl/common/manager/rpc_job_process.py

```diff
@@ -57,6 +57,12 @@ def _acceptor(self, job_type):
                     continue
                 raw_job = json.loads(job.value.decode('utf-8'))
                 try:
+                    # If current node's id does not fall under job's node_ids,
+                    # ignore the job and dont process
+                    if "node_ids" in raw_job:
```

Comment:
Its not a mandatory parameter and if its present then we check if current node's id falls under this list.